source/nextor_sys/Makefile

```
@@ -32,20 +32,20 @@ define assemble_as
 	@$(N80) $(1) $(2) $(3)
 endef
 
-nextor-sys: NEXTOR.SYS NEXTORK.SYS
+nextor-sys: NEXTOR.SYS NEXTOR.SYS.japanese
```

checkmake (0.3.2)

[warning] Target "nextor-sys" should be declared PHONY.

(phonydeclared)